### DISCUSSION TITLE
Automatic menu generation using text to speech engine

### DISCUSSION THREAD

**Discussion by Nuvzz** [marian-m12l/studio#237] [Ideas]
While making custom stories using Studio, I had an idea on how to improve menu creation.

It would be useful to have a feature that can read the title or text of the "boxes" in Studio web UI (menu, choice, etc..) and generate an audio file (mp3 or ogg) and automatically insert it in the pack.

A user would then have to simply type expected menu selection text and Studio would do the rest ;-)

Some parameters could be introduced in the settings menu to enable or disable this feature and maybe chose the speech engine (male, female, voice type etc.)

**Comment by jersou** [comment]
Hi, here is a project that may interest you, it is not integrated with Studio, but it can generate packs for Studio.
It automatically generates images and sounds for menus using folder/file names.
→ https://github.com/jersou/studio-pack-generator

**Reply by benjapan** [reply]
Personaly, I use Voicemaker to generate questions and audio titles : https://voicemaker.in/
Don't forget to add a silence before sound like this : <break time='0.5s'/ Hello Lunii break time='0.5s'/<break time='500ms'/

**Reply by Nuvzz** [reply]
Thank you and this is helpful too
